controllers/login.controller.es6

```diff
@@ -31,6 +43,51 @@ export default class LoginController {
     );
   }
 
+  toggleInfo() {
+    this.showInfo = !this.showInfo;
+  }
+
+  connectLedger() {
+    this.ledgerStatus = 'Not connected';
+    new StellarLedger.Api(new StellarLedger.comm(20))
+      .connect(() => {
+        this.ledgerStatus = 'Connected';
+        this.$scope.$apply();
+      }, () => {
+        this.ledgerStatus = 'Error: ' + err;
+        this.$scope.$apply();
+      });
+  }
+
+  proceedWithLedger() {
+    this.bip32Path;
+    try {
+      new StellarLedger.Api(new StellarLedger.comm(5)).getPublicKey_async(this.bip32Path).then((result) => {
```

Comment:
Minor issue and not in scope of this PR, but I'd change the name to `getPublicKeyAsync`.

Reply:
I was following the convention established here: https://github.com/LedgerHQ/ledger-node-js-api/tree/master/src

Reply:
OK, if it's idiomatic to Ledger, let's leave it as it is. Thanks!